core/src/main/java/org/jboss/pnc/causeway/ctl/ImportControllerImpl.java

```diff
@@ -75,7 +75,7 @@ public ImportControllerImpl() {
     @Override
     @Asynchronous
     public void importBuild(Build build, CallbackTarget callback, String username) {
-        logger.log(Level.INFO, "Entering importBuild.");
+        logger.log(Level.INFO, "Importing build {0} to tag {1}", new Object[]{build.getBuildName(), build.getTagPrefix()});
```

Comment:
It would be good to switch to slf4j and use varargs and not constructing array of objects. This can wait for next versions.

Reply:
More information about the build should be logged - buildID or URL to build